### ISSUE TITLE
Tools page: Remove entry for IBM Dynamic Assessment Plugin

### ISSUE THREAD

**Issue by maryjom** [w3c/wai-website#256]
[Web Accessibility Evaluation Tools List](https://www.w3.org/WAI/ER/tools/) has an old tool link:

The information and link to IBM DAP - Dynamic Assessment Plugin can be removed since you've added links to the IBM Equal Access Accessibility Checker (our latest, freely available tool).  I know, the link still works on our site and I'm trying to get the team to remove/redirect it. 

This entire entry can be removed:
> IBM DAP - Dynamic Assessment Plugin for Chrome Developer Tools by IBM Accessibility Research
> High-performance plugin integrates automated accessibility checking with developer tools. DAP quickly scans modern web applications, identifies accessibility issues, and recommends potential fixes so you understand how to correct violations. Covers WCAG 2.0 conformance, include WAI-ARIA markup.
> 
> http://www-03.ibm.com/able/dynamic-assessment-plug-in.html, Released: 2017-Mar-09
> 
> Detailed Information about “IBM DAP - Dynamic Assessment Plugin for Chrome Developer Tools”

**Comment by shawna-slh** [comment]
Submitted to maintaining list: https://lists.w3.org/Archives/Public/public-wai-ert-tools/2021May/0001.html
